### PR TITLE
[CLI]Build docker image with CLI

### PR DESCRIPTION
**Pull request by ashishb**
### Description

1. Build docker image with CLI and geth
2. Add missing dependencies
3. Change git+ssh to git+https for some yarn dependencies given that the
code is now open-source
4. Fix a few more minor bugs in the process

### Tested

Build command:
```
docker build -f dockerfiles/cli/Dockerfile.cli -t gcr.io/celo-testnet/celocli:$USER . --build-arg celo_env=alfajores
```

Run container:
```
docker rm cli_container; docker run -it bash  -p 8545:8545 gcr.io/celo-testnet/celocli:$USER
```

Run commands against it:
```
docker exec -it cli_container /celocli <sub-command>
```

### CODE REVIEW

File: packages/cli/start_geth_docker.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# TODO(ashishb): testing only
+ls /usr/local/bin
+
+# Usage: start_geth_docker.sh [geth binary location] [network name] [sync mode]
+# Delete this
+GETH_BINARY=${1:-"/usr/local/bin/geth"}
+# Default to testing the integration network
```
Comment (yerdua) [inline packages/cli/start_geth_docker.sh]: nit: this comment doesn't match the default set below, which is "alfajores"

File: packages/cli/start_geth_docker.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+NETWORK_NAME=${2:-"alfajores"}
+# Default to testing the full sync mode
```
Comment (yerdua) [inline packages/cli/start_geth_docker.sh]: nit: this also doesn't match. the default set below is "ultralight"

File: packages/cli/start_geth_docker.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+SYNCMODE=${3:-"ultralight"}
+# Default to 1101
+NETWORK_ID=${4:-1101}
```
Comment (yerdua) [inline packages/cli/start_geth_docker.sh]: Isn't the network id for alfajores `44781`?

File: packages/cli/start_geth_docker.sh (hunk 1 above)
Comment (ashishb) [reply packages/cli/start_geth_docker.sh]: Comment fixed

File: packages/cli/start_geth_docker.sh (hunk 2 above)
Comment (ashishb) [reply packages/cli/start_geth_docker.sh]: Comment fixed.

File: packages/cli/start_geth_docker.sh (hunk 3 above)
Comment (ashishb) [reply packages/cli/start_geth_docker.sh]: Changed to 44781

**Pull request merged**